src/controllers/cartController.js

```diff
@@ -0,0 +1,61 @@
+import {Product, cart} from "../database/models";
+
+class CartController {
+  static async addItem(req, res) {
+    try {
+      const {productId} = req.params;
+      const product = await Product.findOne({ where: { productId, available: true } });
+      if (!product) {
+        return res.status(404).json({ status: 404, message: "Product not found" });
```

Comment:
you are duplicating statuscode
try
res.status(404).json({ status: "fail", message: "Product not found" });